src/languages/fr-FR.js

```diff
@@ -0,0 +1,119 @@
+const { Language, util } = require('klasa');
+
+module.exports = class extends Language {
+
+	constructor(...args) {
+		super(...args);
+		this.language = {
+			DEFAULT: (key) => `${key} n'a pas encore été traduit en 'fr-FR'.`,
+			SETTING_GATEWAY_EXPECTS_GUILD: 'Le paramètre <Guild> attend soit un id soit une instance de Guild.',
+			SETTING_GATEWAY_VALUE_FOR_KEY_NOEXT: (data, key) => `La valeur '${data}' pour la clef '${key}' n'existe pas.`,
+			SETTING_GATEWAY_VALUE_FOR_KEY_ALREXT: (data, key) => `La valeur '${data}' pour la clef '${key}' existe déjà.`,
+			SETTING_GATEWAY_SPECIFY_VALUE: 'Vous devez spécifier une clef pour ajouter ou filtrer.',
+			SETTING_GATEWAY_KEY_NOT_ARRAY: (key) => `La clef '${key}' n'est pas un Array.`,
+			SETTING_GATEWAY_KEY_NOEXT: (key) => `La clef '${key}' n'existe pas dans le schema de data actuel.`,
+			SETTING_GATEWAY_INVALID_TYPE: 'Le paramètre \'type\' doit être soit \'add\' ou \'remove\'.',
+			SETTING_RESOLVER_INVALID_USER: 'Cette clef attend un id ou une instance de User.',
```

Comment:
All SETTING_RESOLVER strings were removed in some updates since this pr was opened.